src/scripts/components/Tooltip.jsx

```diff
@@ -85,8 +87,8 @@ class Tooltip extends React.Component {
 
     $tooltip.setAttribute('id', this.tooltipId);
     $tooltip.setAttribute('role', 'tooltip');
-    $tooltip.classList.add('tooltip');
-    $tooltipContent.classList.add('tooltip-inner', `tooltip-inner--${this.props.type}`);
+    $tooltip.className += 'tooltip';
+    $tooltipContent.className += `tooltip-inner tooltip-inner--${this.props.type}`;
```

Comment:
As discussed with juan, he suggested to add classes separately by classList.add like below
```suggestion
     $tooltipContent.classList.add('tooltip-inner');
     $tooltipContent.classList.add(`tooltip-inner--${this.props.type}`);
```

Also add comment before these lines why we are doing this